src/report-filter-controls.tsx

```diff
@@ -40,9 +25,11 @@ function ReportFilterControls({ path }: ReportFilterControlsProps): VNode {
       return;
     }
     const formData = Array.from(new FormData(form)) as string[][];
-    const seachParams = new URLSearchParams(formData);
-    Object.entries(overrideFormData).forEach(([key, value]) => seachParams.set(key, String(value)));
-    route(`${path}?${seachParams.toString().replace(/\+/g, "%20")}`);
+    const searchParams = new URLSearchParams(formData);
```

Comment:
We could use `setParameters` here as well? My thinking with it was also that we could avoid the need for a separate `src/url-params.ts` altogether, since the `pathWithParams` function could either be inlined into `defaultSetParameters` or at least in the `src/context/report-filter-context.tsx` file.

Reply:
Gave that a shot in 48d97c2a72c2390a52749f80a60c417ff99db13c and f778d06ef73cd23d836601265e1f1e41f0968879